### PR TITLE
remove erroneous " " character

### PR DESCRIPTION
**Pull request by jspencer12**
switched "s3://edx-course-data/{org}/" from "s3://edx-course- data/{org}/"

## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jspencer12! I've created [OSPR-3024](https://openedx.atlassian.net/browse/OSPR-3024) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by openedx-webhooks** [comment]
@jspencer12 Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-documentation&entry.867055334=https://github.com/edx/edx-documentation/pull/1797&entry.1484655318=https://github.com/jspencer12&entry.752974735=2019-01-29+22:13&entry.1917517419=2019-01-29+22:13&entry.2133058324=No) so we can improve your experience in the future.

File: en_us/data/source/access/download.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Follow these steps to access your edX data package on Amazon S3.
 #. To access the database data files, specify or select ``s3://course-data``.
 
    To access the event data files, specify or select ``s3://edx-course-
-   data/{org}/``. You must include the identifier for your organization after
+  data/{org}/``. You must include the identifier for your organization after
```
Comment (grantgoodmanedX) [inline en_us/data/source/access/download.rst:35]: This indentation isn't working with our Travis build. Try putting the line break after select in line 34

**Comment by natabene** [comment]
@jspencer12 Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions.

**Comment by jspencer12** [comment]
-Jonathan Spencer

**Comment by natabene** [comment]
@jspencer12 Have you had a chance to sign the agreement? If so, can you please send it over?

**Comment by jspencer12** [comment]
See attached

**Comment by natabene** [comment]
@jspencer12 We have not heard from you in a while, so I will close this PR for now. Should you decide to pursue this again, please feel free to re-open it and send your signed agreement to our legal team.

**Comment by openedx-webhooks** [comment]
@jspencer12 Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-documentation&entry.867055334=https://github.com/edx/edx-documentation/pull/1797&entry.1484655318=https://github.com/jspencer12&entry.752974735=2019-01-29+22:13&entry.1917517419=2019-04-08+19:39&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**